### PR TITLE
i18n(pl): translate import.mdx

### PR DESCRIPTION
**Pull request by karolhas**
#### Description (required)

Translate Import.mdx page to Polish.

Discord username: HSK (*hskdaking*)

#### Related issues & labels (optional)

- Suggested label: i18n
- Hacktoberfest: hacktoberfest-accepted

### CODE REVIEW

File: src/content/docs/pl/guides/imports.mdx [hunk 1]
```diff
@@ -0,0 +1,285 @@
+---
+title: Importowanie
+description: Naucz się jak importować różne typy treści za pomocą Astro.
+i18nReady: true
+---
+import RecipeLinks from "~/components/RecipeLinks.astro";
+import ReadMore from '~/components/ReadMore.astro'
+
+Astro obsługuje większość statycznych zasobów bez konieczności jakiejkolwiek konfiguracji. Możesz użyć instrukcji `import` w dowolnym miejscu w JavaScript Twojego projektu (w tym w sekcji frontmatter Astro), a Astro uwzględni zbudowaną, zoptymalizowaną kopię tego statycznego zasobu w Twojej finalnej kompilacji. `@import` jest również obsługiwany wewnątrz CSS i tagów `<style>`.
```
[the remark this answers is not in the document]
Comment (karolhas) [reply src/content/docs/pl/guides/imports.mdx]: Done ✅

**Comment by yanthomasdev** [comment]
Thanks @karolhas, we got two small new commits in this page, could you apply them? https://github.com/withastro/docs/commits/main/src/content/docs/en/guides/imports.mdx

**Comment by karolhas** [comment]
Yeah sure

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work!

**Pull request merged**